.env.base

```diff
@@ -41,7 +41,7 @@ REACT_APP_FEATURE_CUSTOM_TOKEN_IMPORT=true
 REACT_APP_FEATURE_ARBITRUM_BRIDGE_CLAIMS=true
 REACT_APP_FEATURE_USDT_APPROVAL_RESET=true
 REACT_APP_FEATURE_PORTALS_SWAPPER=true
-REACT_APP_FEATURE_RUNEPOOL=false
+REACT_APP_FEATURE_RUNEPOOL=true
```

Comment:
@0xApotheosis since LP is happy but lending and RUNE things aren't, can we revert this guy and keep for a follow-up?

Reply:
Discussed this one out with @kaladinlight, going to revert RUNEPool from this diff and land this guy 🙏🏽

Reply:
91276a3ce504e48de83791a0ae3f09ad0c1e531f